core/src/main/resources/error/error-classes.json

```diff
@@ -699,6 +699,13 @@
       }
     }
   },
+  "INVALID_WHERE_CONDITION" : {
+    "message" : [
+      "Found the invalid expressions: <invalidExprs>",
+      "in the where condition: [<condition>]",
+      "Please, rewrite your query and avoid using of the expressions because any window, aggregate or generator expressions are not allowed in where/filter clauses."
```

Comment:
```suggestion
      "The WHERE condition <condition> contains invalid expressions: <expressionList>."
      "Rewrite the query to avoid window functions, aggregate functions, and generator functions in the WHERE clause."
```